### PR TITLE
Update forkgram-telegram from 4.5.5 to 4.5.6

### PR DESCRIPTION
**Pull request by miccal**
Created with `brew bump-cask-pr`.

### CODE REVIEW

**Review by bevanjkay — DISMISSED** [review]

**Review by samford — COMMENTED** [review]
It appears that upstream hasn't provided the usual macOS zip files in the newest 4.5.7 release, hence the mismatch with the `livecheck` version. In this scenario, we used to check all the assets on the release page to find the newest release that has the files we need. Unfortunately, GitHub doesn't include release asset lists in the page HTML anymore (it's fetched separately using JavaScript now), so we can't easily do this anymore.

In this case, the only way to effectively identify the newest release with the file we need is to fetch the assets list HTML for each release tag until we find a match. Worst case scenario, we would waste 11 requests if there aren't any matches (e.g., if no releases provide the files or the filename changes such that the regex doesn't match anymore), so we should set a sane maximum.

I've pushed a commit with a `livecheck` block that implements this approach, which would use five requests at most (one to fetch the releases page and a maximum of four release asset list fetches). There are definitely other casks with a similar situation but I'm not sure that we want to just copy this approach for those.

GitHub has said that we might be better off using the API for this sort of thing but I haven't looked into whether that would actually work for us. Off hand, we've specifically avoided using the GitHub API in livecheck because hitting the rate limit would cause every other GitHub API check to fail in a given run, not to mention rate limit failures being a pain on CI (we had this problem back in the day in the homebrew-livecheck repository before I did away with the GitHub API checks).

Long story short, this works for now but I think we should do some more research before working on checks for similar casks. If this ends up being the only usable approach in the end, it would be better to create a strategy for it (e.g., so we simply can use something like `strategy :github_release_assets`) instead of duplicating this lengthy `strategy` block.

**Comment by miccal** [comment]
Thank you @samford.

**Review by p-linnane — APPROVED** [review]
